packages/scene-composer/src/hooks/useMatterportTags.ts

```diff
@@ -2,9 +2,32 @@ import { useCallback } from 'react';
 
 import { useSceneComposerId } from '../common/sceneComposerIdContext';
 import { IAnchorComponent, ISceneNode, KnownComponentType } from '../interfaces';
-import { IAnchorComponentInternal, ISceneNodeInternal, useStore } from '../store';
+import { IAnchorComponentInternal, IDataOverlayComponentInternal, ISceneNodeInternal, useStore } from '../store';
 import { MattertagItem, TagItem } from '../utils/matterportTagUtils';
 import { RecursivePartial } from '../utils/typeUtils';
+import { Component } from '../models/SceneModels';
+import { generateUUID } from '../utils/mathUtils';
+
+const getContentForOverlayComponent = (label: string, description: string): string => {
+  return `#### **${label}**  
+${description}`;
```

Comment:
this indentation looks weird.. can we just add `\n` to make new line?

Reply:
This is the preferred way from Markdown guideline.

Reply:
also is that possible for the item to not have label and/or description? so we may only have one line or even no overlay at all?

Reply:
Label will always be there.

Reply:
Label is similar to the Tag name.

Reply:
Added a comment in code